File: python/chapter3/shrinking_guest_list.py
guest_list = ["Melkey", "Johnny Depp", "Linus Trovalds"]

print(f"Dear {guest_list[0]}, you are invited to my party!!")
print(f"Dear {guest_list[1]}, you are invited to my party!!")
print(f"Dear {guest_list[2]}, you are invited to my party!!")

print(f"\nUnf{guest_list[2]}tunately %s can't make it.\n")

guest_list = ["Melkey", "Johnny Depp", "Rich Harris"]
print(f"Dear {guest_list[0]}, you are invited to my party!!")
print(f"Dear {guest_list[1]}, you are invited to my party!!")
print(f"Dear {guest_list[2]}, you are invited to my party!!")

print("\nI found a bigger table for the guests!!\n")
guest_list.append("The Primegean")
guest_list.append("Wagslane")

print("\nSorry guys, I can only invite 2 guests!!\n")
guest_list.pop()
guest_list.pop()
guest_list.pop()

print(guest_list)

File: go/chapter3/shrinking_guest_list.go
package main

import "fmt"

func main() {
	guestList := []string{"Melkey", "Johnny Depp", "Linus Trovalds"}

	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[0])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[1])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[2])

	fmt.Printf("\nUnfortunately %s can't make it.\n\n", guestList[2])

	guestList = []string{"Melkey", "Johnny Depp", "Rich Harris"}
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[0])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[1])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[2])

	fmt.Printf("\nI found a bigger table for the guests!!\n\n")
	guestList = append([]string{"The Primegean"}, guestList...)
	guestList = append(guestList, "Wagslane")

	fmt.Printf("\nSorry guys, I can only invite 2 guests!!\n\n")
	guestList = append(guestList[:2])
	fmt.Println(guestList)
}
